ShopGo/HomeScreen/HomeViewController.swift

```diff
@@ -8,24 +8,28 @@
 import UIKit
 
 class HomeViewController: UIViewController {
-
+    
+    @IBOutlet weak var containerView: UIView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        containerView.isHidden = true
     }
-
+    
     @IBAction func trendingButtonTapped(_ sender: Any) {
     }
     
     @IBAction func burgerMenuButton(_ sender: Any) {
+        openAndHideContainerView()
     }
     
-    @IBAction func womenButtonTapped(_ sender: Any) {
+    @IBAction func menButtonTapped(_ sender: Any) {
     }
     
-    @IBAction func childrenButtonTapped(_ sender: Any) {
+    @IBAction func womenButtonTapped(_ sender: Any) {
     }
     
-    @IBAction func sideMenu(_ sender: Any) {
+    @IBAction func childrenButtonTapped(_ sender: Any) {
     }
```

Comment:
You changed the names of these button, just to make sure, can you write a print statement in each to check if it crashes or not

Reply:
@zuhair5 Also made some changes in this file, ping him and check if it will cause conflicts then revert these changes

Reply:
> You changed the names of these button, just to make sure, can you write a print statement in each to check if it crashes or not

Buttons are working fine and the app is not crashing.